### ISSUE TITLE
Organisation of documentation website

### ISSUE THREAD

**Issue by ZedThree** [Unidata/netcdf-c#2566]
There's a lot of very useful information in the documentation, but it can be _very_ difficult to find. A few separate issues as examples:

-For example, there is a "Tutorial" section, but the [page for "Creating new files"](https://docs.unidata.ucar.edu/netcdf-c/current/creating.html) doesn't actually explain how to create new files, or point to where that information can be found.

The tutorial section is buried in the table of contents below "In-memory support", "NCZarr Implementation", and two appendices.

This makes it challenging to find somewhere to point beginners at to learn how to start using netCDF.

**Comment by WardF** [comment]
Agreed.  We are doing our best to get it cleared up, but the ever-present resource allocation issues are a roadblock.  I'm working on getting things cleaned up a bit for the `4.9.1` release, but it won't be the complete overhaul we'd like to see.
